# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/moz.build
# Commit: af4c24767e48
# Full Hash: af4c24767e48def595991abacc5719eb77e1b27f
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 1
# Description:
#   Bug 1654112 - Make mediapipeline_unittest pass and remove TaskQueueWrapper::MainAsCurrent. r=ng,bwc
#   
#   This patch makes mediapipeline_unittest pass an event target to the Call wrapper
#   but keeps using main thread for that event target, so that it can keep the test
#   cases simpler by allowing sybnc calls into the conduits.
# ==============================================================================

diff -r d621668762f4 -r af4c24767e48 dom/media/webrtc/libwebrtcglue/moz.build
--- a/dom/media/webrtc/libwebrtcglue/moz.build	Wed Apr 07 15:01:03 2021 +0200
+++ b/dom/media/webrtc/libwebrtcglue/moz.build	Fri Apr 09 10:26:12 2021 +0200
@@ -21,7 +21,6 @@
     "GmpVideoCodec.cpp",
     "MediaConduitInterface.cpp",
     "MediaDataCodec.cpp",
-    "TaskQueueWrapper.cpp",
     "VideoConduit.cpp",
     "VideoStreamFactory.cpp",
     "WebrtcCallWrapper.cpp",